src/OpenSage.Game/Diagnostics/MainView.cs

```diff
@@ -138,8 +138,10 @@ public void Draw(ref bool isGameViewFocused)
                             {
                                 var random = new Random();
                                 var playableSides = _context.Game.GetPlayableSides().ToList();
-                                var faction1 = playableSides[random.Next(0, playableSides.Count())];
-                                var faction2 = playableSides[random.Next(0, playableSides.Count())];
+                                //var faction1 = playableSides[random.Next(0, playableSides.Count())];
+                                //var faction2 = playableSides[random.Next(0, playableSides.Count())];
+                                var faction1 = playableSides[3];
```

Comment:
Why is that change?

Reply:
because for testing it is a pain in the ass when the faction is randomly chosen :)